pkgs/os-specific/linux/tiscamera/default.nix

```diff
@@ -65,12 +76,12 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = pythonPath;
 
   cmakeFlags = [
-    "-DBUILD_ARAVIS=OFF" # For GigE support. Won't need it as our camera is usb.
-    "-DBUILD_GST_1_0=ON"
-    "-DBUILD_TOOLS=ON"
-    "-DBUILD_V4L2=ON"
-    "-DBUILD_LIBUSB=ON"
-    "-DBUILD_TESTS=ON"
+    "-DTCAM_BUILD_ARAVIS=OFF" # For GigE support. Won't need it as our camera is usb.
```

Comment:
@lovesegfault  :

I have something at https://github.com/jraygauthier/nixpkgs/commit/859886912142cc2ebd58b894a3c8285786598270 that builds. However, latest nixpkgs has aravis v0.8 whereas this package require aravis v0.6 so I was forced to re-introduce v0.6. Having to maintain 2 versions of a package in nixpkgs is not something that is generalized favored.

I know that tiscamera v1 moved to aravis v0.8 so it will be possible then

It think it would warrant either warrant another PR that re-introduce aravis v0.6 or even better just wait until I get a fully working and tested tiscamera v1 (currently undergoing work here to make sure everything still works fine with v1).

Note also that my current camera (from a client) is usb so other than ensuring that it builds I cannot test the aravis ethernet support so this is not something I can be fully maintainer of. If somebody else owning such a a camera wants to add himself as maintainer for the gige aravis backend aspect, that would be nice.

Reply:
> or even better just wait until I get a fully working and tested tiscamera v1

I'm fine with that option, thank you for the explanation!

Reply:
I own GigE TIS cameras, but they're somewhere in the middle of my ongoing move

Reply:
See #178139 for v1 PR.